website/docs/d/is_endpoint_gateway_targets.html.markdown

```diff
@@ -0,0 +1,31 @@
+---
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : egts"
+description: |-
+  Fetches endpoint gateway targets information.
+---
+
+# ibm\_egts
+
+Import the details of endpoint gateway targets on IBM Cloud as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+
+    data "ibm_is_endpoint_gateway_targets" "endpointGatewayTargets" {
+    }
+
+```
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
```

Comment:
There are no arguments here..
So can this be just.. `The following attributes are exported:`